hormonal-hedonists/README.md

```diff
@@ -1,15 +1,30 @@
 # Hormonal Hedonists
+## Project name: The Legend of Archimedes
+![archimedes](/hormonal-hedonists/assets/images/archimedes.jpg)
 
-Please use this README to document your team's project. Make sure to include a general description, information on how to set-up and run your project, and anything else you think may be interesting for a README. The README is usually the first document people read when they visit a project on GitHub, so it's a good idea to make it appealing.
+##### Installing dependencies
 
-## Useful links
+```
+python -m pip install requirements.txt
+```
+or
+```
+pip install pipenv
+pipenv install Pipfile
+```
 
-- **Code Jam Rules:** [https://pythondiscord.com/pages/code-jams/code-jam-6/rules/](https://pythondiscord.com/pages/code-jams/code-jam-6/rules/)
+## Running the project
+`pipenv run start`
```

Comment:
This section doesn't cover the `pip` installation method listed above in the "Installation" section.